admin/tabs/general/roles/RolePanel.ts

```diff
@@ -42,7 +42,18 @@ export const rolePanel = hoistCmp.factory({
                     }),
                     filterChooser({flex: 1})
                 ],
-                item: hframe(vframe(grid(), roleGraph()), detailsPanel())
+                item: hframe(
+                    vframe(
+                        grid({
+                            agOptions: {
```

Comment:
I don't think this is necessary anymore - perhaps left over from old implementation?